grg_pssedata/struct.py

```diff
@@ -983,7 +994,7 @@ def validate(self):
     def to_psse(self):
         '''Returns: a pss/e encoding of this data structure as a string'''
         data = [self.p1, self.p2, self.w1, self.w2]
-        return os.linesep.join([x.to_psse() for x in data])
+        return '\n'.join([x.to_psse() for x in data])
```

Comment:
This change is interesting, I would expect it to be less robust. Can you elaborate on this one?

Reply:
There are some troubles to write the file in text mode. In Windows u get a lot of blanklines because ```os.linesep``` introduces ```\r\n``` at the end of the line. But it is interpreted as a double line break. See the [Docs Recomendations](https://docs.python.org/3/library/os.html#os.linesep).

An small example:
``` 
case = grg_pssedata.io.parse_psse_case_file('case3.raw')
f = open('foo.raw', 'w')
f.write(case.to_psse())
f.close()
``` 
With the ```master``` branch:
```
0, 100.0, 33, 0.0, 0.0, 60.0

tests basic network fetures

this case does not have an HVDC line that occurs in the Matpower case

  1, '1           ', 240.0, 3, 1, 1, 1, 1.10000002, 0.0, 1.1, 0.9, 1.1, 0.9

  2, '2           ', 240.0, 2, 1, 1, 1, 0.92616999, 7.25883, 1.1, 0.9, 1.1, 0.9

  3, '3           ', 240.0, 2, 1, 2, 1, 0.89999998, -17.267101, 1.1, 0.9, 1.1, 0.9

0 / END OF BUS DATA, BEGIN LOAD DATA
```

With the ```ICSEG-cases``` branch:

```
0, 100.0, 33, 0, 0, 60.0
tests basic network fetures
this case does not have an HVDC line that occurs in the Matpower case
  1, '1           ', 240.000, 3, 1, 1, 1, 1.10000, 0.00000e+00, 1.10000, 0.90000, 1.10000, 0.90000
  2, '2           ', 240.000, 2, 1, 1, 1, 0.92617, 7.25883, 1.10000, 0.90000, 1.10000, 0.90000
  3, '3           ', 240.000, 2, 1, 2, 1, 0.90000, -17.267, 1.10000, 0.90000, 1.10000, 0.90000
0 / END OF BUS DATA, BEGIN LOAD DATA
```

Reply:
Got it, I had a miss understanding of how python uses `os.linesep` vs `'\n'`, these changes look good to me.